rclcpp/test/rclcpp/test_publisher.cpp

```diff
@@ -506,6 +506,6 @@ TEST_F(TestPublisher, run_event_handlers) {
 
   for (const auto & handler : publisher->get_event_handlers()) {
     std::shared_ptr<void> data = handler->take_data();
-    EXPECT_THROW(handler->execute(data), std::runtime_error);
```

Comment:
If the test was passing before for FastRTPS, does that mean it was throwing an exception here, or that there were no event handlers and this loop was never entered?

Reply:
> there were no event handlers and this loop was never entered?

Exactly that.
The only "default" event handler is the "incompatible qos" event, which is not implemented in fastrtps.